### DISCUSSION TITLE
Meaning of MaxKeys variable

### DISCUSSION THREAD

**Discussion by dthemg** [minio/minio#14797] [Q&A] [answered]
Hi, and thanks for this wonderful project

I created a bucket for uploading my users profile pictures, but i noticed that there is a MaxKeys variable:
![image](https://user-images.githubusercontent.com/13008236/164648621-5d13a83f-d2e9-495e-bd52-41d730c9df4d.png)

Does this variable indicate the max number of objects in the bucket? If so, how can I go about to increase it? My intention was to have all users' (100k+) profile pictures in the same bucket, since it simplifies creation of the download links, but maybe this is a bad idea.

Best regards

David

**Comment by klauspost** [comment, marked as answer]
@dthemg This is the maximum number of keys returned in this listing result.

https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html#API_ListObjectsV2_ResponseSyntax

**Reply by dthemg** [reply]
Got it! Thanks for answering
